src/lib/modernExtend.ts

```diff
@@ -1464,11 +1465,17 @@ export function actionEnumLookup(args: ActionEnumLookupArgs): ModernExtend {
             if (attributeKey in msg.data) {
                 let value = getFromLookupByValue(msg.data[attributeKey], lookup);
                 // endpointNames is used when action endpoint names don't overlap with other endpoint names
-                if (args.endpointNames) value = postfixWithEndpointName(value, msg, model, meta);
+                if (args.endpointNames) {
+                    value = addEndpointName(value, msg, model, meta, args.actionTemplate === 'action_endpoint' ? 'postfix' : 'prefix');
```

Comment:
For consistency purposes, I would always postfix it

Reply:
But what to to with existing backwards implementations then?

Reply:
It's only used by `tuyaOnOffAction` -> `SBDV-00032` (new device), so we don't have to consider this yet?

Reply:
> It's only used by `tuyaOnOffAction` -> `SBDV-00032` (new device), so we don't have to consider this yet?

All other similar Tuya based buttons have this structure

Reply:
@Koenkk tuyaOnOffAction is really just modern version of tuyaFz.on_off_action

Should I just revert all changes with postfix/prefix and use old tuyaFz.on_off_action instead?

Reply:
Could we have a `tuyaOnOffAction` and `tuyaLegacyOnOffAction`?

Reply:
> Could we have a `tuyaOnOffAction` and `tuyaLegacyOnOffAction`?

Sounds reasonable

Reply:
Done